ccd-definition/CaseEventToFields/WorkAllocationIntegrationFields-SDO.json

```diff
@@ -73,6 +73,19 @@
     "DisplayContext": "MANDATORY",
     "Publish": "Y"
   },
+  {
+    "CaseTypeID": "CIVIL",
```

Comment:
Remember to move this field to WorkAllocationIntegrationFields-SDO-CUI-nonprod.json